Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
        sys.modules["bpython"] = {}
        color = _Color(3, True)
>       assert len(color.wrap_text("test")) == 16
E       AssertionError: assert 12 == 16
E        +  where 12 = len('\x1b[3mtest\x1b[0m')
E        +    where '\x1b[3mtest\x1b[0m' = wrap_text('test')
E        +      where wrap_text = [3mcolor[0m.wrap_text

../test.py:9: AssertionError
=========================== short test summary info ============================
FAILED ../test.py::test - AssertionError: assert 12 == 16
============================== 1 failed in 0.03s ===============================
